tests/test_multipart.py

```diff
@@ -255,14 +256,13 @@ def test_read_chunk_properly_counts_read_bytes(self):
         self.assertEqual(b'.' * size, result)
         self.assertTrue(obj.at_eof())
 
-    def test_read_does_reads_boundary(self):
+    def test_read_does_not_read_boundary(self):
```

Comment:
Well, actually test reads the boundary (: In previous case it just was placed into unread buffer.